examples/arkane/species/C2H6/input.py

```diff
@@ -11,6 +11,7 @@
 useHinderedRotors = True
 useBondCorrections = False
 author = 'I.B. Modeling'
+frequencyScaleFactor = 0.99
```

Comment:
This is a very confusing point:
The **ZPE** scaling factor for CBS-QB3 is indeed 0.99, but the **frequency** scaling factor (which Arkane requires) for CBS-QB3 is 0.99 * 1.014 =~ 1.004. I know we've all been using 0.99, we only standardized this entire scaling factors  thing recently. The good news is that it's automated, so we don't need to worry about it too much. I suggest either not adding this line, or assign `0.99 * 1.014` (the latter is more transparent).

(The 1.014 is a factor between ZPE scaling and freq. scaling that is universal, according to Truhlar - almost independent of the level of theory)

Reply:
I think I'll go with your second suggestion and modify this line and the one above it to:
'''
# Example of how to manually set the frequencyScaleFactor
frequencyScaleFactor = 0.99*1.014  # 0.99 for CBS-QB3, 1.014 for scaling between ZPE and freq. according to Truhlar
'''

The reason I wanted to keep this line was because it is the only species example where we manually set the frequencyScaleFactor. Does this seem reasonable? If so I'll make this change and force push it

Reply:
The ref for CBS-QB3's **ZPE** scaling factor is [dx.doi.org/10.1063/1.477924](https://dx.doi.org/10.1063/1.477924), and the ref for the 1.014 factor is [dx.doi.org/10.1021/ct100326h](https://dx.doi.org/10.1021/ct100326h)

Reply:
Thanks! I'll add this and push the changes